Jenkinsfile

```diff
@@ -25,6 +27,34 @@ node {
 				sh './scripts/cibuild'
 			}
 		}
+
+		env.OAR_SETTINGS_BUCKET = 'openapparelregistry-staging-config-us-east-1'
+
+		if (env.BRANCH_NAME == 'develop' || env.BRANCH_NAME.startsWith('release/') || env.BRANCH_NAME.startsWith('test/')) {
+			// Publish container images built and tested during `cibuild`
+			// to the private Amazon Container Registry tagged with the
+			// first seven characters of the revision SHA.
+			stage('cipublish') {
+				// Decode the ECR endpoint stored within Jenkins.
+				withCredentials([[$class: 'StringBinding',
+								credentialsId: 'OAR_AWS_ECR_ENDPOINT',
+								variable: 'OAR_AWS_ECR_ENDPOINT']]) {
+					wrap([$class: 'AnsiColorBuildWrapper']) {
+						sh './scripts/cipublish'
+					}
+				}
+			}
+
+			// Plan and apply the current state of the instracture as
+			// outlined by the `develop` branch of the `open-apparel-registry`
```

Comment:
It looks like this comment is inaccurate. Instead of using the Terraform configuration on `develop`, it'll use the configuration of whatever branch passed the conditional above (`develop`, `release/*`, `test/*`).